### ISSUE TITLE
As Vaults user, I need an informative landing page when I first navigate to vaults UI

### ISSUE THREAD

**Issue by otoole-brendan** [Agoric/dapp-inter#1]
## What is the Problem Being Solved?

We need to implement a new Vaults landing page that serves as a informational jumping off point for users to proceed in creating a vault. It's intended to explain to users at a glance what vaults are/what they can do and funnel them to clear 'Create Vault' CTA.

## Description of the Design

-Reference Figma design https://www.figma.com/file/r0WcCjyS0LAOalFGV4DSuk/agoric-v1?node-id=1876%3A14385&t=M1K9MKcyPF9GJnCO-0 (although note Ekko need to convey the tiles arent clickable)

- [ ] footer links including Inter Discord, docs, Twitter etc.
- [ ] clickable analytics button which brings users to analytics.inter.trade

## Security Considerations
Unknown

## Test Plan
Smoke Test upon implementation

**Comment by otoole-brendan** [comment]
@turadg @samsiegart  The purpose of this page has evolved - it is not just to show vault type previews, it serves as informative vault landing page where we they can learn at a glance what they can do/what vaults are and click a CTA button to start creating a vault. In short it's more of a marketing/informative jumping off point so should not be blocked by us not supporting multiple vault types at launch.
I'd like to remove the dependency so that is clear, I will also rename this issue so it's clearer.

**Comment by otoole-brendan** [comment]
Edited ticket title per my comment

**Comment by otoole-brendan** [comment]
@samsiegart I have added more specific detail/requirements based on latest Ekko design - some are not MVP critical, I have marked as such

**Comment by otoole-brendan** [comment]
Note: in an effort to cut scope, this view has been deferred. Instead, users will land on My Vaults Dashboard page. General page requirements have been moved to this ticket https://github.com/Agoric/dapp-inter/issues/12

**Comment by otoole-brendan** [comment]
Not needed
